src/inputReader/YAMLReader.cpp

```diff
@@ -0,0 +1,86 @@
+#include "YAMLReader.h"
+
+#include <spdlog/spdlog.h>
+#include <yaml-cpp/yaml.h>
+
+#include <array>
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+#include <vector>
+
+#include "../LinkedCells/Cell.h"
```

Comment:
Don't use relative include paths!!! Use as root what you put in the variable `target_include_directives`. These relative includes are OS dependant and may break!